src/Filter/types/index.ts

```diff
@@ -342,9 +343,15 @@ export interface RadioFilterProps {
  * meant to override default text filter behaviour.
  */
  export interface SingleSelectFilterOptions extends Omit<Partial<FilterModule<string>>, 'description' | 'label' | 'required'> {}
- 
+
  /**
  * `TextFilterOptions` is any valid `FilterModule` property (excluding description and label)
  * meant to override default text filter behaviour.
  */
  export interface TextFilterOptions extends Omit<Partial<FilterModule<string>>, 'description' | 'label'> {}
+
```

Comment:
nit: Would be nice to keep this in alpha order too, though this of course would only be seen by developers searching through the types. As an aside (not something you should do), but we might consider splitting up this types folder into more separated types and just exporting everything from this index file now that its growing so much. What do you think?

Reply:
Done. Alphabetized for now. We can organize this better when we do some refactor work to better organize the components into their own folder. Let's add some issues for it.